agent.go

```diff
@@ -723,7 +723,7 @@ func (s *sandbox) listenToUdevEvents() {
 		fieldLogger.Infof("Received add uevent")
 
 		// Check if device hotplug event results in a device node being created.
-		if uEv.DevName != "" && strings.HasPrefix(uEv.DevPath, rootBusPath) {
+		if uEv.DevName != "" && strings.HasPrefix(uEv.DevPath, rootBusPath) && uEv.SubSystem == "block" {
```

Comment:
I dont think we should add this constraint for "block" devices. This was added to be generic enough. I know today we use it for block devices only, but we could expand it to other kind of devices in the future.

Reply:
Yeah, I had removed it.